extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/sse/SseResource.java

```diff
@@ -93,8 +92,8 @@ public void sseJson2(Sse sse, SseEventSink sink) throws IOException {
 
     @Path("json/multi")
     @GET
-    @Produces(MediaType.SERVER_SENT_EVENTS)
     @RestStreamElementType(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.SERVER_SENT_EVENTS)
```

Comment:
Why is it required here?

Reply:
Ah, it got left over. I will fix it later on today